lib/SIL/SILPrinter.cpp

```diff
@@ -1693,6 +1693,12 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     *this << ", ";
     *this << AMI->getType();
   }
+  void visitCXXVirtualMethodInst(CXXVirtualMethodInst *CXXVMI) {
+    *this << getIDAndType(CXXVMI->getOperand()) << ", " << CXXVMI->getMember();
+    *this << " : " << CXXVMI->getMember().getDecl()->getInterfaceType();
+    *this << ", ";
+    *this << CXXVMI->getExtractedMethod()->getType();
+  }
```

Comment:
Any updates to the SIL parser to go with this?

Reply:
Yes, this prints out in a form equivalent to ObjCMethodInst and can piggyback on the parsing already done by ObjCMethodInst.